examples/misc/bin/try_dart/functions.dart

```diff
@@ -0,0 +1,22 @@
+// A function declaration.
+int timesTwo(int x) {
+  return x * 2;
+}
+
+// Arrow syntax is shorthand for `{ return expr; }`.
+int timesFour(int x) => timesTwo(timesTwo(x));
+
+// Functions are objects.
+int runTwice(int x, Function f) {
+  for (var i = 0; i < 2; i++) {
+    x = f(x); // ignore: invalid_assignment
```

Comment:
would using a local variable make this warning go away?

Reply:
No, but giving `f` the type of `int Function(int)` works.